Freelancer/DATA/EQUIPMENT/engine_equip.ini

```diff
@@ -7672,3 +7898,217 @@ outside_cone_attenuation = -3
 inside_sound_cone = 60
 outside_sound_cone = 180
 lootable = true
+
+;; Winter Event
+
+[Engine]
+nickname = santa_engine_fighter_heavy01
+ids_name = 459027
+;res str
+; SANTA’S ENGINE ($class)
+ids_info = 460649
+;res html
+; \m\bSANTA VON CLAUSSEN’S ENGINE Engine\B

```

Comment:
It says "ENGINE engine". These duplications should be removed. Same for the following engine infocards